src/main.jsx

```diff
@@ -1,10 +1,12 @@
-import React from 'react'
-import ReactDOM from 'react-dom/client'
-import App from './App.jsx'
-import './index.css'
+// eslint-disable-next-line no-unused-vars
```

Comment:
It's not a good practise to disable linters, kindly resolve the issue that the linter is pointing out. Thank you. 🙏